about.html

```diff
@@ -0,0 +1,126 @@
+                   <a href="index.html">
+                       <div class="logo">
+                           <p>Techsphere</p>
+                           <i class="fa fa-globe fa-3x" aria-hidden="true"></i>
+                       </div>
+                   </a>
+                   <ul class="topBarUl">
+                       <li><a id="about" href="about.html">About</a></li>
+                       <li><a href="index.html#">Program</a></li>
+                       <li><a href="#">Join</a></li>
+                       <li><a href="#">Sponsor</a></li>
+                       <li><a href="#">News</a></li>
+                   </ul>
+                   <button class="topBarBtn">Campaign</button>
+               </div>
+           </nav>
+       </section>
+       <div class="aboutIntroContainer">
+           <nav class="aboutNav mobile popup">
+               <span class="rect rec1"> </span>
+               <span class="rect rec2"> </span>
+               <span class="rect rec3"> </span>
+           </nav>           
+           <p class="aboutHello">"Hello World!"</p>
+           <h1 class="aboutIntroH1">Techsphere Global <br>Summit 2022</h1>
+           <p class="aboutTechSphere">The Techsphere Global Summit brings together the community of experts, academics, activists and enthusiasts who comprise the bustling tech space. This year, we hope to expand our invitation list including organizations and individuals</p>
+           <p class="contactInfo">Please contact us via Email for any further enquiries about Techsphere Global Summit 2022!</p>
+           <a class="contactEmail" href="#">techspheresubmit22@gmail.com</a>
+       </div>
+   </section> 
+
+   <section class="logoSection">
+       <div class="logoContainer">
+           <h1>Techsphere Global Summit 2022 Logo</h1>
+           <hr>
+           <p>The logo of Techsphere Global Summit 2022 was decided through the logo competition from 8 June to 7 July</p>
+           <div class="logo">
+               <p>Techsphere</p>
+               <i class="fa fa-globe fa-5x" aria-hidden="true"></i>
+           </div>
+       </div>
+   </section>
+
+   <section class="memories">
+       <div class="memoriesContainer">
+           <h1>See the past Techsphere Global Summits</h1>
+           <hr>
+           <p>Take a look at the last two Techsphere Global Summits which took place in Ghana and in Kenya</p>
+           <div class="memoriesDiv1">
+               <a class="memorieslink" href="#">
+                   <div class="memoriesDiv">
+                       <h1>2020</h1>
+                       <p>Techsphere Global Summit in Ghana</p>
+                   </div>
+               </a>
+               <a class="memorieslink" href="#">
+                <div class="memoriesDiv">
+                    <h1>2018</h1>
+                    <p>Techsphere Global Summit in Kenya</p>
+                </div>
+            </a>
+           </div>           
+       </div>
+   </section>
```

Comment:
- [ ] Kindly increase the margins to fit Cindy's design as stated in the imagebelow:

| Your Design |
|---------------|
|![image](https://user-images.githubusercontent.com/62938735/190866762-59dfbad0-72c0-4d11-b3c4-6fd3c8d4101d.png)|